.github/workflows/build.yaml

```diff
@@ -0,0 +1,63 @@
+name: "Build"
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+  linux-jvm-build:
+    name: Linux - JVM build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ 17 ]
+        jvm-builder-image: [ 'paketobuildpacks/builder:tiny' ]
```

Comment:
Will it be possible to declare different builder images to verify if they can be used to build the container ? The motivation behind this request is because soon it will be needed to test the paketo UBI builder image and later in a next iteration our red hat builder image